src/index.html

```diff
@@ -1,13 +1,41 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img src="./images/imac.jpeg" alt="imac"
+      class="container__image">
+      <h2 class="container__headline">Моноблок APPLE A1419 iMac 27&quot;  Retina 5K (MNED2UA/A)</h2>
+      <div class="container__code">Код товара: 195434</div>
+      <div class="container__revies">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="container__count-review">Отзывов: 5</div>
+      </div>
+      <div class="container__cost-info">
+        <p class="container__cost">Цена:</p>
+        <p class="container__how-cost">69 999 грн</p>
+      </div>
+      <div class="container__button">
+        <a href="#buy" data-qa="hover"
+        class="call-back-button">
```

Comment:
```suggestion
        <a href="#buy" data-qa="hover"
        class="call-back-button">
```
Checklist about 3 attributes in one tag!